backend/database/migrations/2023_01_15_034134_add_role_to_admins_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('admins', function (Blueprint $table) {
+            $table->string('role')->default('1');
+        });
+    }
```

Comment:
What is the use of role? 
Is the role naa sa ERD?
I also noticed that Admin is separated table while ERD is not? Can you explain this?

Reply:
Nachanged sir while doing it and I prefer to use separated User and Admin sir kay naglibug ko unsaon pag implement. 🙇